### PR TITLE
Add perf test for RabbitMQ.Client

### PR DESCRIPTION
**Pull request by mikeharder**

### CODE REVIEW

**Comment by mikeharder** [comment]
@HarshaNalluru, @ramya-rao-a: Ready for review

**Review by weshaggard — APPROVED** [review]

**Review by jsquire — APPROVED** [review]
Outside the scope of these changes, but the directory structure under `tests`  seems strange.   Maybe we should consider a sibling for `performance-tests` or another container for unit/functional tests?

File: sdk/servicebus/Microsoft.Azure.ServiceBus/tests/Performance/Microsoft.Azure.RabbitMQ.Client.Performance/Program.cs [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+                    ExecuteReceives(channel, entityPath, messages);
+                }
+                else
+                {
+                    ExecuteSends(channel, entityPath, messages);
+                }
+            }
+        }
+
+        private static void ExecuteSends(IModel channel, string entityPath, long messages)
```
Comment (HarshaNalluru) [inline sdk/servicebus/Microsoft.Azure.ServiceBus/tests/Performance/Microsoft.Azure.RabbitMQ.Client.Performance/Program.cs]: Looks good.
@mikeharder  Is there a way to know the number of accepted and rejected messages like we do in the rhea-promise test ?

**Review by HarshaNalluru — APPROVED** [review]

File: sdk/servicebus/Microsoft.Azure.ServiceBus/tests/Performance/Microsoft.Azure.RabbitMQ.Client.Performance/Program.cs (hunk 1 above)
Comment (HarshaNalluru) [reply sdk/servicebus/Microsoft.Azure.ServiceBus/tests/Performance/Microsoft.Azure.RabbitMQ.Client.Performance/Program.cs]: https://github.com/mikeharder/azure-sdk-for-js/blob/9d21c89adff6dacff2a520283f6f196e37fbb716/sdk/servicebus/service-bus/test/perf/rhea-promise/send.ts#L102

**Comment by pakrym** [comment]
This PR has been open for a while. Can we merge it or close it?

**Comment by mikeharder** [comment]
@pakrym: I'm not sure why this wasn't merged, maybe I forgot about it.  I will review this week and either merge or close.

**Comment by pakrym** [comment]
Closing this PR. Please reopen if it still needs to be merged.

**Pull request closed without merging**